TOC.md

```diff
@@ -26,6 +26,7 @@
       - [跨数据中心部署方案](dev/how-to/deploy/geographic-redundancy/overview.md)
       - [配置集群拓扑](dev/how-to/deploy/geographic-redundancy/location-awareness.md)
     - [使用 Ansible 部署 DM 集群](dev/how-to/deploy/data-migration-with-ansible.md)
+    - [TiDB-Binlog 集群部署](dev/how-to/deploy/tidb-binlog-deploy.md)
```

Comment:
I suggest **change the filename into `tidb-binlog.md`** because in the URL we don't need another `deploy`. @dcalvin What do you think?

```suggestion
    - [部署 TiDB Binlog 集群](dev/how-to/deploy/tidb-binlog.md)
```

Reply:
Agree.